### ISSUE TITLE
problem with new scipy version

### ISSUE THREAD

**Issue by sazabi4** [jobovy/galpy#340]
I think this line needs to be changed now, as scipy is at v1.1.0

https://github.com/jobovy/galpy/blob/b9eb44dffcd128e2ba3104c03757be7da15d25f9/galpy/orbit_src/FullOrbit.py#L8

**Comment by jobovy** [comment]
What is the problem you are having? I thought I had edited that line already for >v1... Does the code parse the version incorrectly? I think
```
_SCIPY_VERSION[0] should be 1
```
and
```
_SCIPY_VERSION[1] should be 1
```
such that this condition fails and logsumexp is taken from the correct module?

**Comment by sazabi4** [comment]
Sorry... I noticed the problem in galpy 1.2 (from pip install) but I did not notice that this has been corrected in the latest version on the GitHub version. I should have read the code in GitHub when I open the issue. My bad....

**Comment by jobovy** [comment]
No problem; this should also already be fixed in v1.3 which pip should install now.
